src/client/styles/components/_layout.scss

```diff
@@ -19,11 +20,141 @@ fieldset {
 	padding-bottom: 10px;
 }
 
+//desktop pad-top 30 but 4 is from header leine height
 .page-section {
-	padding: 20px 0px 80px 0px;
-
+	@include media(mobile){
+		padding: 18px 5px 80px 0px;		//top: 18 + 2(header line ht)=20
+	}
 	@include media(desktop){
-		padding: 30px 0px 80px 0px;
+		padding: 26px 5px 80px 0px;   		//top :26+4(header line ht)=30
 	}
     
+}
+
+// left margin for back link in mobile
+.button--back{
+	@include media(mobile){
+	margin-left:14px;
+	
+	}
+	
+}
+
+// .page-section class from frontend-uk over written in order to do spacing changes
+.page-section * + *{
+	margin-top:0px;
+}
+
+.form-title{
+
+	@include media(desktop){
+		font-weight:bold;
+		padding-bottom:30px; // 40= 30+4px(line ht header)+6px line ht of (title)form-fields
+		line-height: 44px;
+	}
+
+	@include media(mobile){
+		font-weight:bold;
+		font-size:24px;
+		line-height:29px;
+		padding-bottom:25px; //30 =26 + 2px(line ht)+3 form-fields
+	}
+}
+
+.form-textbox, .form-dropdown{
+	@include media(desktop){
+		//margin-top:15px;
+		margin-top:9px;
+	}
+
+	@include media(mobile){
+		//margin-top:10px;
+		margin-top:6px;
+	}
+}
+
+.multiple-choise-horizontal, .multiple-choise-vertical{
```

Comment:
These look like modifiers. Would be better to use the correct BEM syntax for modifiers: `multiple-choice--horizontal` or maybe `multiple-choice--inline`.